app/models/document.rb

```diff
@@ -50,9 +50,17 @@ class Document < ActiveRecord::Base
   validates_presence_of :project, :title, :category
   validates_length_of :title, :maximum => 60
 
-  scope :visible, lambda {|*args| { :include => :project,
-                                    :conditions => Project.allowed_to_condition(args.first || User.current, :view_documents) } }
-  scope :with_attachments, includes(:attachments).where("attachments.container_id is not NULL" )
+  scope :visible, -> {
+    includes(:project)
+      .where(Project.allowed_to_condition(User.current, :view_documents))
+      .references(:projects)
+  }
+
+  scope :with_attachments, -> {
+    includes(:attachments)
+      .where("attachments.container_id is not NULL" )
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Space inside parentheses detected.